api/pom.xml

```diff
@@ -53,7 +53,6 @@
     <description>Aggregator Module for Artefacts that exposes the public API of Payara Application Server</description>
 
     <modules>
-        <module>payara-api</module>
         <module>payara-bom</module>
```

Comment:
This change needs to be reverted

Reply:
Ok thanks. This make sense. So i will revert this change this evening.